.gitignore

```diff
@@ -1,6 +1,8 @@
 /dist
 /node_modules
+/result
 
-# Legacy paths
-/.publish
-/package-lock.json
+# This is a `bun` project, but we still use `package-lock.json` as the source
+# of truth for our dependencies.
+# See the `bun.lockb` target in `Makefile` for more details.
+/bun.lockb
```

Comment:
Should we also exclude `/bun.lock` proactively?

Reply:
Sounds good!